wire-ios-sync-engine/Source/UserSession/ZMUserSession.swift

```diff
@@ -530,6 +531,7 @@ public class ZMUserSession: NSObject {
     }
 
     func createMLSClientIfNeeded() {
+        // TODO: [jacob] refactor out
```

Comment:
Is there a Jira ticket reference for this TODO?

Reply:
No I'll create one

Reply:
Done